docs/_data/docstoc.yml

```diff
@@ -64,15 +64,15 @@
  
 - title: 'Deploying Codewind remotely'
   children:   
-    - title: 'Overview'
+    - title: 'Overview: Deploying Codewind remotely'
       url: remote-overview.html
     - title: 'Deploying your remote Codewind instance'
       children:
-      - title: 'Deploying all-in-one'
+      - title: 'Deploying remote Codewind all-in-one'
         url: remotedeploy-combo.html
-      - title: 'Deploying Codewind components individually'
+      - title: 'Deploying remote Codewind components individually'
```

Comment:
the title in the file is "Deploying Codewind components individually for a Codewinid remote deployment" 

so one of the two has to change

Reply:
see it here: https://github.com/eclipse/codewind-docs/blob/master/docs/_documentations/remotedeploy-single.md